docs/CONFIG-PROPERTIES.md

```diff
@@ -44,6 +44,9 @@
 | newlog.gzipfiles.ondisk.maxmegabytes | integer in Mbytes | 2048 | the quota for keepig newlog gzip files on device |
 | process.cloud-init.multipart | boolean | false | help VMs which do not handle mime multi-part themselves |
 | edgeview.authen.jwt | edgeview session jwt token | empty string(edgeview disabled) | format as standard JWT for websocket session for temporary testing, this configitem will be removed once controllers are setup to send EdgeViewConfig in configuration |
+| netdump.enable | boolean | true | enable publishing of network diagnostics (as tgz archives to /persist/netdump) |
+| netdump.topic.publish.interval | integer in seconds | 1 hour pre-onboarding, 1 day post-onboarding | how frequently (in seconds) can be netdumps of the same topic published |
```

Comment:
If this is set to 2 hours will it make both the pre-onboarding and post-onboarding be two hours? 
Would it make sense to have two separate settings for the two?

Reply:
When setting a config property, you are either doing it inside the config partition of the EVE installer (global.json or bootstrap.pb), which is in effect only has until onboarding and first `/config` request, or you are changing config from the controller and that will be applied after onboarding. So it is effectively not possible to change both at the same time.

Reply:
If I set in (to 10 minutes) in bootstrap.pb and when it is onboarded and the EdgeDevConfig from the controller contains no ConfigItem for netdump.topic.publish.interval, what interval will be used? 10 minutes? Or do the semantics imply that it switches to 1 day when controller says nothing?  
Whatever the behavior it will be hard to explain without listing 4-6 examples.

Reply:
Fair point, I have split it to:
`netdump.topic.preonboard.interval` and
`netdump.topic.postonboard.interval`
(removed `publish` to keep the config param names at reasonable length)